Assets/Shopify/UIToolkit/Shops/Generic/Classes/GenericCartView.cs

```diff
@@ -0,0 +1,53 @@
+namespace Shopify.UIToolkit {
+    using UnityEngine;
+    using UnityEngine.UI;
+    using Shopify.Unity;
+    using Shopify.Unity.SDK;
+    using System.Collections.Generic;
+    using System;
+
+    public class GenericCartView : MonoBehaviour {
```

Comment:
I think `CartView` and `GenericViewBase or something` exist already, mind merging this with those classes?